rct229/ruletest_engine/ruletest_engine.py

```diff
@@ -257,7 +258,7 @@ def run_section_tests(test_json_name):
 
         # Evaluate rule and check for invalid RMRs
         evaluation_dict = evaluate_rule(rule, rmr_trio)
-        # pprint.pprint(evaluation_dict)
+        pprint.pprint(evaluation_dict)
```

Comment:
Did you intent to leave this uncommented?

Reply:
I commented it out. Thanks for catching it